sonar/theme/templates/sonar/partial/navbar.html

```diff
@@ -58,7 +43,7 @@
           class="d-inline-block align-top mr-3 my-4" alt="">
       </a>
       {% if not admin %}
-      <form action="{{ url_for('documents.search', view=view_code) }}"
+      <form action="{{ url_for('documents.search', view=view_code if g.get('organisation', {}).get('isDedicated') else config.SONAR_APP_DEFAULT_ORGANISATION) }}"
```

Comment:
you should add a default return to `False` for the `get('isDedicated')`. If the `get('organisation', {})` return an empty dict, then `get('isDedicated')` will failed.

Reply:
I think it's not correct, this instruction will never fail. If the organisation is an empty object, `isDedicated` is resolved as `None` which is the same behavior than `False` in this case. What do you think about it?

Here's a quick example: https://repl.it/@sebastiendeleze/test

Reply:
I'm agree ; it's just not really human readable.